akvo/rsr/views/my_rsr.py

```diff
@@ -29,7 +29,7 @@ def my_details(request):
 
     profile_form = ProfileForm(
         initial={
-            'email': request.user.email,
+            'emailform': request.user.email,
```

Comment:
Might be a mistake on my part. Hade to debug and later realized it was in the Ajax url. Will review when I can

Sent from my iPhone

> On 12 Jan 2015, at 11:55, Kasper Brandt <notifications@github.com> wrote:
> 
> In akvo/rsr/views/my_rsr.py:
> 
> > @@ -29,7 +29,7 @@ def my_details(request):
> >  
> >      profile_form = ProfileForm(
> >          initial={
> > -            'email': request.user.email,
> > +            'emailform': request.user.email,
> Why this change?
> 
> —
> Reply to this email directly or view it on GitHub.
>